ai/smart-wallet-example/smart-wallet-core-solidty/contracts/erc20/MyERC20Token.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
```

Comment:
I think most of the contract except OTKAccount.sol can be removed?

Reply:
I removed all of the non relevant scripts.